Conda build

Progression at: https://github.com/jorgensd/staged-recipes-mpc/tree/dokken/adios4dolfinx

Most tests pass locally.
Currently ipyparallel tests fail with:
```bash
___________ test_read_write_3D_vector_non_simplex[4-NCF-False-False] ___________

write_mesh = False, family = 'NCF', degree = 4, is_complex = False
create_non_simplex_mesh_3D = PosixPath('/tmp/pytest-of-conda/pytest-0/output11/original_mesh_3D_non_simplex.xdmf')
cluster = <ipyparallel.client.client.Client object at 0x7fc86e847ca0>
get_dtype = <function get_dtype.<locals>._get_dtype at 0x7fc86e545700>
tmp_path = PosixPath('/tmp/pytest-of-conda/pytest-0/test_read_write_3D_vector_non_7')

    @pytest.mark.skipif(MPI.COMM_WORLD.size > 1, reason="Test uses ipythonparallel for MPI")
    @pytest.mark.parametrize("write_mesh", [True, False])
    @pytest.mark.parametrize("is_complex", [True, False])
    @pytest.mark.parametrize("family", ["NCF"])
    @pytest.mark.parametrize("degree", [1, 4])
    def test_read_write_3D_vector_non_simplex(
        write_mesh, family, degree, is_complex, create_non_simplex_mesh_3D, cluster, get_dtype, tmp_path
    ):
        fname = create_non_simplex_mesh_3D
    
        f_dtype = get_dtype(np.float64, is_complex)
    
        def f(x):
            values = np.empty((3, x.shape[1]), dtype=f_dtype)
            values[0] = np.full(x.shape[1], np.pi) + x[0]
            values[1] = np.cos(x[2])
            values[2] = x[0]
            if is_complex:
                values[2] += x[0] * x[1] * 3j
            return values
    
        query = cluster[:].apply_async(
            write_function_vector, write_mesh, fname, family, degree, f, f_dtype, "u_original", tmp_path
        )
        query.wait()
>       assert query.successful(), query.error
E       AssertionError: [<RemoteError[ ]:ModuleNotFoundError(No module named 'test_original_checkpoint')>, <RemoteError[ ]:ModuleNotFoundError(No module named 'test_original_checkpoint')>]
E       assert False
E        +  where False = <bound method AsyncResult.successful of <AsyncResult(write_function_vector): failed>>()
E        +    where <bound method AsyncResult.successful of <AsyncResult(write_function_vector): failed>> = <AsyncResult(write_function_vector): failed>.successful

tests/test_original_checkpoint.py:500: AssertionError
```

--- a/README.md
+++ b/README.md
@@ -2,6 +2,7 @@
 
 ![MIT](https://img.shields.io/github/license/jorgensd/adios4dolfinx)
 [![status](https://joss.theoj.org/papers/7866cb142db8a803e32d79a109573d25/status.svg)](https://joss.theoj.org/papers/7866cb142db8a803e32d79a109573d25)
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/adios4dolfinx/badges/version.svg)](https://anaconda.org/conda-forge/adios4dolfinx)
 
 ADIOS4DOLFINx is an extension for [DOLFINx](https://github.com/FEniCS/dolfinx/) to checkpoint meshes, meshtags and functions using [ADIOS 2](https://adios2.readthedocs.io/en/latest/).
 
@@ -21,12 +22,13 @@ The `adios4dolfinx` library extends the [DOLFINx](https://github.com/FEniCS/dolf
 
 Compatibility with DOLFINx:
 
-- ADIOS4DOLFINx v0.7.3 is compatible with DOLFINx v0.7.x
 - ADIOS4DOLFINx v0.8.1 is compatible with DOLFINx v0.8.x
+- ADIOS4DOLFINx v0.7.3 is compatible with DOLFINx v0.7.x
 
 ### Dependencies
 
-The library depends on the Python-interface of [DOLFINx](https://github.com/) and an MPI-build of [ADIOS2](https://adios2.readthedocs.io/en/latest/setting_up/setting_up.html#as-package) and can therefore not be installed through PyPi, but has to be installed through Conda, Spack or from source.
+The library depends on the Python-interface of [DOLFINx](https://github.com/) and an MPI-build of [ADIOS2](https://adios2.readthedocs.io/en/latest/setting_up/setting_up.html#as-package).
+Therefore `ADIOS2` should not be install through PYPI/pip, but has to be installed through Conda, Spack or from source.
 
 ### Docker
 
@@ -70,16 +72,17 @@ Following is a minimal recipe of how to install adios4dolfinx, given that you ha
 ```bash
 conda create -n dolfinx-checkpoint python=3.10
 conda activate dolfinx-checkpoint
-conda install -c conda-forge fenics-dolfinx pip adios2=*=mpi_*
-python3 -m pip install adios4dolfinx[test]@git+https://github.com/jorgensd/adios4dolfinx@v0.8.1
+conda install -c conda-forge adios4dolfinx
 ```
 
 > [!NOTE]
-> To run the tests or demos associated with the code, install `ipyparallel` in your environment, for instance by calling
->
-> ```bash
-> python3 -m pip install adios4dolfinx[test]@git+https://github.com/jorgensd/adios4dolfinx@v0.8.1
-> ```
+> Remember to download the appropriate version of `adios4dolfinx` from Github [adios4dolfinx: Releases](https://github.com/jorgensd/adios4dolfinx/releases)
+
+To run the test suite, you should also install `ipyparallel`, `pytest` and `coverage`, which can all be installed with conda
+
+```bash
+conda install -c conda-forge ipyparallel pytest coverage
+```
 
 ## Functionality
 

